Analysis/HiggsTauTau/src/HTTWeights.cc

```diff
@@ -92,8 +92,9 @@ namespace ic {
     em_qcd_cr2_2to4_          = nullptr;
     em_qcd_cr2_gt4_           = nullptr;
     ele_tracking_sf_          = nullptr;
-    muon_tracking_sf_          = nullptr;
-    scalefactor_file_           = "";
+    muon_tracking_sf_         = nullptr;
+    scalefactor_file_         = "";
+    do_tau_id_sf_             = false;
```

Comment:
do_tau_id_sf_ already exists a few lines before this one, should probably remove one of the two.